web-animations-1/Overview.bs

```diff
@@ -3960,9 +4111,164 @@ interface Animation : EventTarget {
     method unpauses the animation and, if the animation has already finished
     playing in the reversed direction, seeks to the start of the <a>target
     effect</a>.
+:   <dfn method for=Animation lt='persist()'>void persist()</dfn>
+::  Sets this animation's [=replace state=] to
+    [=persisted replace state|persisted=].
+:   <dfn method for=Animation lt='commitStyles()'>void commitStyles(properties)</dfn>
```

Comment:
Do you expect that only applying some of the animation styles but not all of them will be a common use case? Might it make sense to start with an API where all properties are applied and allow for upgrading in the future?

Reply:
Yes, I'm not sure. I added this at @graouts' suggestion but I'm fine with leaving it off and adding it later if that's preferable. (And when I implemented it I made this feature a separate patch to prove it could be incrementally added.)

Reply:
I think we should start without any arguments and consider adding a list of properties in a future version if we feel it is warranted.